starter-workflows/README.md

```diff
@@ -0,0 +1,12 @@
+## Starter Workflows
+
+The Frogbot Code Scanning workflow is configured in this directory.
+Two files can be edited: frogbot-scan-pr.yml, which configures the scan workflow, and forgbot-fix.yml, which opens a new
+Pull Request with the necessary changes if a vulnerability is found.
+To start using Starter-Workflows please refer the GitHub documentation:
```

Comment:
To start using Starter-Workflows please refer the GitHub documentation:
-->
To start using Starter-Workflows, please refer to the [Using starter workflows
GitHub documentation](https://docs.github.com/en/actions/using-workflows/using-starter-workflows).

Reply:
done